sample/src/main/java/ru/terrakok/cicerone/sample/utils/moxy/MvpAppCompatActivity.java

```diff
@@ -0,0 +1,68 @@
+package ru.terrakok.cicerone.sample.utils.moxy;
+
+import android.annotation.SuppressLint;
+import android.os.Bundle;
+import androidx.appcompat.app.AppCompatActivity;
+import com.arellomobile.mvp.MvpDelegate;
+
+@SuppressLint({ "Registered", "unused" })
+public class MvpAppCompatActivity extends AppCompatActivity {
```

Comment:
Why do you copy-pasted this class? Jetifier don't work?

Reply:
My bad, fixed it.